bin/cli.js

```diff
@@ -158,6 +158,10 @@ if (opts.slow) {
   mocha.slow(opts.slow)
 }
 
+if (opts.require) {
+  require(fs.absolute(opts.require));
+}
+
```

Comment:
Yeah I think that would be best until we could match the same `--require` functionality. Actually I'd call it `--modules` as not confuse the user that we would be loading them in the browser side (we are loading them on the phantomjs side)